public/locales/fr/videocall.json

```diff
@@ -22,15 +22,15 @@
     },
     "errors": {
         "title": "Une erreur est survenue",
-        "desc": "Une erreur est survenue qui vous empêchera probablement de voir votre proche, voici le détail:",
+        "desc": "Une erreur est survenue, l'appel ne peut aboutir. Voici le détail:",
         "errors": {
-            "error-1655267250": "The call you're trying to join does not exist or has expired (after 2 hours).",
-            "error--1494481250": "The call you're trying to join does is no longer available (after 2 hours).",
-            "error-1189215202": "No internet available or stable.",
-            "error-763909797": "An unknown error happened during the call.",
-            "error--140619965": "The call is now locked (too much people or expired after 2 hours).",
-            "camera-error--621623973": "The camera or microphone has not been allowed.",
-            "camera-error--1412228987": "The camera or microphone reported an error (hardware or software issue on your device)."
+            "error-1655267250": "L'appel que vous avez rejoint a expiré (après 2 heures).",
+            "error--1494481250": "L'appel que vous avez rejoint n'est plus disponible (après 2 heures).",
+            "error-1189215202": "Connexion interrompu, veuillez vérifier votre connexion internet.",
+            "error-763909797": "Une erreur inconnue est survenue",
+            "error--140619965": "L'appel est vérouillé (trop de personne l'on rejoins ou il a expiré, en vrai j'en sais rien)",
```

Comment:
Daily may be able to provide more info on the cause of this error, so users can be better informed?

In the meantime, here is a suggestion and some corrections for this message : "L'appel est verrouillé, soit parce qu'il a expiré, soit en raison d'un nombre excessif de participants."